lib/symbioticpy/symbiotic/environment.py

```diff
@@ -42,21 +42,13 @@ def _set_symbiotic_environ(tool, env, opts):
     llvm_prefix = '{0}/llvm-{1}'.format(env.symbiotic_dir, llvm_version)
 
     if not isdir(llvm_prefix):
-        dbg('Did not find a build of LLVM, checking the system LLVM')
+        dbg('Did not find a build of LLVM, checking install/ directory')
+        llvm_prefix = '{0}/install/llvm-{1}'.format(env.symbiotic_dir, llvm_version)
+        env.prepend('PATH', '{0}/bin'.format(llvm_prefix))
         if not _check_clang_in_path(llvm_version):
-            dbg("System's LLVM does not have the right version ({0})".format(llvm_version))
-            dbg("Cannot use system LLVM neither the directory with LLVM binaries exists: '{0}'".format(llvm_prefix))
```

Comment:
I am not sure about this patch. Without looking for system tools, Symbiotic will not work without the `install/` folder when built using  `system-build.sh`. I do not think we want that. Using stuff from `install/` folder while the scripts are in the devel mode was always really just a fallback solution.

Reply:
I see. What about using `opt` to find LLVM directory? The problem is that we can't just blindly use `clang` from `PATH` as it may be a symbolic link to some compiler wrapper like `ccache` and in that case `which clang` may not be the same as `which opt`.

Reply:
The problem as actually even more complicated on Ubuntu if you don't use unversioned tools, e.g. you have `opt-9` instead of `opt` in `PATH`.

Reply:
I've pushed a patch that should fix these problems in the CI but we may still encounter them on our machines.

Reply:
Is this PR ready to go, or you plan to add anything?

Reply:
Sorry, I missed this comment. Yeah, the PR's ready, but the issue mentioned in https://github.com/staticafi/symbiotic/pull/209#discussion_r771939894 is still present.